tests/subsys/net/lib/nrf_cloud/src/main.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <ztest.h>
+#include <zephyr.h>
+#include <stdio.h>
+#include <net/nrf_cloud.h>
+#include "nrf_cloud_fsm.h"
+
+void test_init_not_idle(void)
```

Comment:
I recommend adding prefix comments to your test-case functions which explicitly describe the specific failure case you are testing, here, as with the other test cases

Reply:
and here as well, thanks